vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MergedRoomCreationItem.kt

```diff
@@ -133,7 +133,10 @@ abstract class MergedRoomCreationItem : BasedMergedItem<MergedRoomCreationItem.H
         val membersCount = roomSummary?.otherMemberIds?.size ?: 0
 
         if (isDirect) {
-            holder.roomDescriptionText.text = holder.view.resources.getString(R.string.this_is_the_beginning_of_dm, roomSummary?.displayName ?: "")
+            holder.roomDescriptionText.text = holder.view.resources.getString(
+                    R.string.this_is_the_beginning_of_dm,
+                    distinctMergeData.lastOrNull()?.memberName ?: ""
```

Comment:
What if the name is changed?

Reply:
The change only concerns the string: "This is the beginning of your direct message history with ... " 
At the end of this string we have to display the other member name and not the roomMember name. (it's the case in the web application).
About which name are you talking ? the member name or the room name ?

Reply:
oh i understand. I think we should show the current name.